services/core-web/src/components/admin/complianceCodes/ComplianceCodeManagement.tsx

```diff
@@ -243,6 +246,13 @@ const ComplianceCodeManagement: FC = () => {
     isEditMode && editModeResetColumn,
   ].filter(Boolean);
 
+  const setExpiredRowBackground = (record: IComplianceArticle) => {
+    if (isAfter(new Date(), new Date(record.expiry_date))) {
```

Comment:
I think that each of the compliance articles should have an is_expired data index

Reply:
You mean to the end of adding that expired column?  I like it, but I think we should leave that for MVP 2.0 personally.

Reply:
Sorry I know this is closed now but I meant that it doesn't have to be calculated again here- each of the entries already has an "is_expired" property.